docker-compose.zap.yaml

```diff
@@ -12,10 +12,10 @@ services:
       - ./test-output:/home/node/test-output
 
   zap-baseline-scan:
-    image: owasp/zap2docker-stable:2.14.0
```

Comment:
Why was the image changed? Is the new one an official image supported by the OWASP project?